shared/chat/conversation/container.js

```diff
@@ -1,181 +1,88 @@
 // @flow
-import * as Constants from '../../constants/chat'
```

Comment:
likely hard to look at this as a diff. The old way we plumbed through a ton of data to fulfill all the possible variants and then used a ton of branch() calls to determine which things to load. Instead we just pull what we need to figure out which type of screen you'll see and delegate to further connected components